suite-common/suite-config/src/fiat.ts

```diff
@@ -15,52 +72,5 @@ export const FIAT = {
         { symbol: 'nmc', coingeckoId: 'namecoin' },
         { symbol: 'zec', coingeckoId: 'zcash' },
     ],
-    currencies: [
-        'usd',
-        'eur',
-        'gbp',
-        'aed',
-        'ars',
-        'aud',
-        'bdt',
-        'bhd',
-        'bmd',
-        'brl',
-        'cad',
-        'chf',
-        'clp',
-        'cny',
-        'czk',
-        'dkk',
-        'hkd',
-        'huf',
-        'idr',
-        'ils',
-        'inr',
-        'jpy',
-        'krw',
-        'kwd',
-        'lkr',
-        'mmk',
-        'mxn',
-        'myr',
-        'nok',
-        'nzd',
-        'php',
-        'pkr',
-        'pln',
-        'rub',
-        'sar',
-        'sek',
-        'sgd',
-        'thb',
-        'try',
-        'twd',
-        'vef',
-        'vnd',
-        'zar',
-        'xdr',
-        'xag',
-        'xau',
-    ],
+    currencies: Object.keys(fiatCurrencies),
```

Comment:
Please remove completely this `FIAT` const, it only used in few places in suite so some refactoring could be done inside of this PR.

Instead currencies export it as `const fiatCurrenciesCodes = Object.keys(fiatCurrencies)` and instead of tickers export it as `tickers` directly.

Reply:
Can you please add JSDocs deprecated comment before this?